README.md

```diff
@@ -22,29 +22,39 @@ At this time, only new issues & pull requests that support the Ionic Framework w
 
 # Local development
 
+This project uses [pnpm](https://pnpm.io/) for package management. You can install it by following the [installation instructions](https://pnpm.io/installation).
+
 ## Getting started
 
 To set up this project and prepare the example project to be used in your own projects run the following commands.
 
 ```bash
-npm install
-npm run bootstrap
-npm run build
+pnpm install
+pnpm build
 ```
 
 This will generate all necessary builds in the example projects. You can then either publish the packages to npm or a private package manager, or use `npm pack` to start using the builds in a local project for testing purposes where you manually place the package in the project `node_modules` folder.
 
 For example, if you have an Angular project, run `npm pack` in `packages/example-project/component-library` and `packages/example-project/component-library-angular`. Since the framework component libraries depend on the Stencil project, you will need to provide the `component-library` package in your project `node_modules` folder.
 
+## Formatting
+
+This project uses [Prettier](https://prettier.io/) for code formatting. You can run the following command to format the code:
+
+```bash
+pnpm prettier
+```
+
 ### Node Versions
 
-This project uses [Volta](https://volta.sh/) to automatically manage what npm and node versions are used when developing. If you don't use Volta, don't worry about it... just check the root `package.json` for what version of node and npm is currently being used.
+This project uses [Volta](https://volta.sh/) to automatically manage what pnpm and node versions are used when developing. If you don't use Volta, don't worry about it... just check the root `package.json` for what version of node and pnpm is currently being used.
```

Comment:
I'm getting that too, if I run `npm run build` or `npm run prettier` from the root

Reply:
I don't know what it is, but something about the previous non-committed or `.gitignore`-ed state in the repo causes this error

I tried a fresh clone of the repo and it worked no problem

```sh
cd /tmp
git clone git@github.com:ionic-team/stencil-ds-output-targets.git
cd ./stencil-ds-output-targets
pnpm install
pnpm build
```

Reply:
so something spooky is going on 👻 

I closed the terminal window I was testing this in and then opened a new one and `cd`-ed into the directory again and now it works no problem...so I think there's something odd about how volta manages `pnpm` when you're switching branches or something?

because when I was initially testing I

1. opened the terminal in `~/`
2. did `cd ~/Code/stencil-ds-output-targets`
3. I had a random feature branch I've been working on checked out
4. did `git fetch origin`
5. then `git checkout sp/pnpm`
6. _then_ I did `pnpm install` and `pnpm run build` and experienced the issue

However then when I left the `sp/pnpm` branch checked out, closed the terminal, and repeated the process of opening a new terminal window, navigating to that directory, and running the same commands it works.

That sort of makes some sense to me, because I think volta runs some stuff automagically when you enter a directory which wouldn't have run for `pnpm` the first time I tried it

Reply:
What's the consensus on the recommended changes to the README here?

Reply:
I think the readme changes suggested by Ryan are probably fine - once this is merged I don't think anyone would run into the weird volta catch-22 related to changing directories anymore so I don't think a code change is needed to address that

Reply:
It sounds like most folks reviewing this PR didn't test/evaluate using Volta, which makes me suspect the value provided by it isn't as strong when both teams aren't fully bought in to it. If using Volta can cause issues with pnpm, I start to under if we should remove Volta from the project.

In another world with more time/resources, I would prefer Volta over pnpm. I don't think that's a realistic choice here, given the circumstances. The problem here is that Volta shims _everything_, making pnpm usage harder for the Stencil team (which we're fully bought into for our other projects we maintain). A temporary workaround might be to use another shell for the Stencil team, but I'm open to other suggestions if anyone has them.